.travis.yml

```diff
@@ -1,8 +1,8 @@
 sudo: false
 language: python
 python:
-    # - "2.7"
     - "3.4"
     - "3.5"
-install: pip install -r requirements.txt && pip install pytest
+    - "3.6"
+install: pip3 install -r requirements.txt && pip3 install pytest
```

Comment:
travis execute `pip` in a virtualenv, `pip` is `pip3` when it's the version selected